apps/aecore/src/aec_blocks.erl

```diff
@@ -116,15 +117,22 @@ serialize_for_network(B = #block{}) ->
     {ok, jsx:encode(serialize_to_map(B))}.
 
 serialize_to_map(B = #block{}) ->
+    Pow =
+        case is_integer(B#block.pow_evidence) of
```

Comment:
I expect this to fail always as that field does not seem integer. I expect this to be list of 42 integers.

Refs:
* https://github.com/aeternity/epoch/wiki/Block-Header
* https://github.com/aeternity/epoch/blob/1fa7a978ddaa897e2bb8cfdd948f2cc1e0e8aac9/apps/aecore/c_src/aec_pow_cuckoo_nif.cpp#L55
* https://github.com/aeternity/epoch/blob/1fa7a978ddaa897e2bb8cfdd948f2cc1e0e8aac9/apps/aecore/test/aec_pow_cuckoo_tests.erl#L60